apps/src/lib/util/firehose.js

```diff
@@ -0,0 +1,114 @@
+/** @file Provides clients to AWS Firehose, whose data is imported into AWS Redshift. */
+
+import AWS from 'aws-sdk';
+
+const IDENTITY_POOL_ID = 'us-east-1:fbfec393-0afb-4682-84d4-59ad04a302f4'
+const AWS_REGION = 'us-east-1'
+
+AWS.config.region = AWS_REGION;
+AWS.config.credentials = new AWS.CognitoIdentityCredentials({
+  IdentityPoolId: IDENTITY_POOL_ID,
+});
+const FIREHOSE = new AWS.Firehose({apiVersion: '2015-08-04'});
+
+/**
+ * A barebones client for posting data to an AWS Firehose stream.
+ * Usage:
+ *   FirehoseClient.putRecord(
+ *     'analysis-events',
+ *     {
+ *       created_at: DateTime.now,           // REQUIRED
```

Comment:
Yeah, that seems reasonable.

It's worth noting that the 'auto' datetime format detection when copying into Redshift can handle a lot of different formats, so we don't necessarily need to pass this through in exactly the Ruby format.